README.md

```diff
@@ -87,91 +108,80 @@ echo -e "admin\n${CONCOURSE_ATC_PASSWORD}" | \
-
- * Use terraform to create the required resources for microbosh.
- * Render `manifests/bosh-manifest` using spruce.
- * Generate a set of random passwords for bosh init.
- * Deploy microbosh using `bosh-init`
+### Destroy
 
-
-To setup the pipeline `destroy-microbosh`:
+Run the `destroy-microbosh` pipeline which can be setup by running:
 
 ```
 ./concourse/scripts/destroy-microbosh.sh $DEPLOY_ENV
 ```
 
-This pipeline will:
+## CloudFoundry
 
- * Get the resources from the deploy microbosh pipeline.
- * Run `bosh-init` to delete the deployment, updating the bosh init state
-   with an empty file.
- * Destroy the resources created by terraform.
+### Prerequisites
 
-### Deploy and destroy cloudfoundry with microbosh
-```
-# Optionally pass the current branch for the git resources
-export BRANCH=$(git rev-parse --abbrev-ref HEAD)
+You will need a working [MicroBOSH](#microbosh).
+
+### Deploy
+
+Run the `deploy-cloudfoundry` pipeline which can be setup by running:
 
+```
 ./concourse/scripts/deploy-cloudfoundry.sh $DEPLOY_ENV
 ```
 
 You can optionally specify a cloudfoundry RELEASE_VERSION (defaults to 225) and
 STEMCELL_VERSION (defaults to 3104) as environment variables.
 
-This pipeline will:
-
- * Get VPC, microbosh and concourse state
- * use this state to run terraform and create required aws resources for cloud
-   foundry
- * build the cloudfoundry manifests
- * use these manifests and microbosh instance to deploy cloudfoundry
- * Setup the cloud controller IAM role to `cf-cloudcontroller`. It must be allowed to access the S3 buckets `*-cf-resources`, `*-cf-packages`, `*-cf-droplets`, `*-cf-buildpacks`.
+NB: This will [auto-delete overnight](#overnight-deletion-of-environments)
+by default.
 
-An additional pipeline `autodelete-cloudfoundry` is setup to automatically
-delete the environment at night.
+### Destroy
 
-To setup destroy pipeline you have to execute:
+Run the `destroy-cloudfoundry` pipeline which can be setup by running:
 
- ```
- ./concourse/scripts/destroy-cloudfoundry.sh <environment_name>
+```
+./concourse/scripts/destroy-cloudfoundry.sh $DEPLOY_ENV
 ```
 
-This pipeline will
+# Additional notes
 
- * Connect to microbosh and delete CF deployment
- * Use Terraform to destroy all the resources created by `deploy-cloudfoundry`
+## Optionally override the branch used by pipelines
 
-# Additional notes
+All of the pipeline scripts honour a `BRANCH` environment variable which
+allows you to override the git branch used within the pipeline. This is
+useful for development and code review:
+
+```
+BRANCH=$(git rev-parse --abbrev-ref HEAD) <pipeline_script>.sh
+```
 
-## SSH tunnel to vagrant concourse-lite and share access to instance
+## SSH to Bootstrap Concourse and sharing
 
 Instead of allowing a non secure HTTP connection via the internet to the
-concourse-lite AWS instance, we create a SSH tunnel to redirect the port
-8080 to localhost.
+*Bootstrap Concourse*, we create a SSH tunnel to redirect the port 8080 to
```

Comment:
It's both: "the local port 8080 to the remote port 8080". Maybe we should omit this altogether?